src/main/java/de/unistuttgart/iste/gits/common/util/PaginationUtil.java

```diff
@@ -0,0 +1,68 @@
+package de.unistuttgart.iste.gits.common.util;
```

Comment:
The three classes in this repo should be moved to a common code repository, as well as the tests